rcl/include/rcl/time.h

```diff
@@ -323,7 +323,7 @@ rcl_fini_duration(rcl_duration_t * duration);
 RCL_PUBLIC
 RCL_WARN_UNUSED
 rcl_time_source_t *
-rcl_get_default_ros_time_source(void);
+rcl_default_ros_time_source_get(void);
```

Comment:
I think this one should be left as-is. Can you undo this particular name change?